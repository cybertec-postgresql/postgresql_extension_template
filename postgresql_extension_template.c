#include "postgres.h"
#include "fmgr.h"

PG_MODULE_MAGIC;

Datum my_function(PG_FUNCTION_ARGS);
PG_FUNCTION_INFO_V1(my_function);

Datum
my_function(PG_FUNCTION_ARGS)
{
	PG_RETURN_CSTRING("Hello from my_function()!");
}
