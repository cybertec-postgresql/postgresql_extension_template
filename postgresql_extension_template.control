# postgresql_extension_template extension
comment = 'Briefly describe your extension here'
default_version = '1.0'
module_pathname = '$libdir/postgresql_extension_template'
relocatable = true
