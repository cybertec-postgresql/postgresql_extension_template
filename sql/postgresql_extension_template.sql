CREATE EXTENSION postgresql_extension_template;

SELECT my_function();
