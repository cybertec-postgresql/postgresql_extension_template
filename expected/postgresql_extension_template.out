CREATE EXTENSION postgresql_extension_template;
SELECT my_function();
        my_function        
---------------------------
 Hello from my_function()!
(1 row)

